cmd/kubeadm/app/master/manifests.go

```diff
@@ -296,6 +296,24 @@ func getAPIServerCommand(cfg *kubeadmapi.MasterConfiguration) []string {
 		"--allow-privileged",
 	)
 
+	if cfg.AuthorizationMode != "" {
+		switch cfg.AuthorizationMode {
+		case "ABAC":
```

Comment:
> Are there no consts for these names anywhere?

https://github.com/kubernetes/kubernetes/blob/master/pkg/kubeapiserver/authorizer/config.go#L33-L39  I don't know if you want to add that to dependency tree, but that's where they live at the moment.